### PR TITLE
Add some CAs

### PR DESCRIPTION
**Pull request by KukiSa**
Domains added to the existing file are of a sub-brand of the company or a subsidiary company.
Domains added to `category-cas` belong to independent WebTrust verified CAs.

ref: https://github.com/threatstop/crl-ocsp-whitelist/blob/8b8bfdeffd6299013e14262799946f9ac1b60782/ocsp-hostnames.txt

### CODE REVIEW

**Comment by KukiSa** [comment]
草，怎么没通过检查，，，

File: data/digicert [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# It is worth noting that although its domain name contains the word cn, it does not actually resolve to servers located in mainland China.
-digicert-cn.com
+digicert-cn.com # Not in Chinese Mainland
```
Comment (rootmelo92118) [inline data/digicert:1]: Should it add the `@!cn` attribute if it does not inside mainland China?

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]

File: data/digicert (hunk 1 above)
Comment (IceCodeNew) [reply data/digicert:1]: That is fine, only the category that mainly focused on sites which have Pops located in China would need the attr `@!cn` .

**Comment by KukiSa** [comment]
`wisekey` is a crypto technology company too, and may also need to be included in other categories.

**Comment by rootmelo92118** [comment]
> `wisekey` is a crypto technology company too, and may also need to be included in other categories.

Maybe you can include it in `category-companies`.

**Comment by IceCodeNew** [comment]
let me have a look

File: data/certinomis [hunk 2]
```diff
@@ -0,0 +1,2 @@
+certinomis.com
+certinomis.fr
```
Comment (IceCodeNew) [inline data/certinomis:2]: 需要注意文件的最后一定要留一个空行。

**Review by IceCodeNew — APPROVED** [review]

File: data/certinomis (hunk 2 above)
Comment (KukiSa) [reply data/certinomis]: 好，谢谢。

**Comment by MkQtS** [comment]
@KukiSa, hello. I just found the whole `yahoo` is included into `category-cas` via the inclusion of `verizon`. It seems not reasonable.

And I found an old news: [DigiCert Acquires Verizon Enterprise SSL Business](https://finance.yahoo.com/news/digicert-acquires-verizon-enterprise-ssl-120000401.html). Could you take a look?

**Comment by KukiSa** [comment]
> @KukiSa, hello. I just found the whole `yahoo` is included into `category-cas` via the inclusion of `verizon`. It seems not reasonable.
> 
> And I found an old news: [DigiCert Acquires Verizon Enterprise SSL Business](https://finance.yahoo.com/news/digicert-acquires-verizon-enterprise-ssl-120000401.html). Could you take a look?

This was my mistake. You may remove the entries related to Verizon.

**Pull request merged**